src/interface/misc.rs

```diff
@@ -53,6 +53,11 @@ pub fn fillzero(bufptr: *mut u8, count: usize) -> i32 {
     for i in 0..count {slice[i] = 0u8;}
     count as i32
```

Comment:
Did you implement this a second time? I think what we want to do is remerge the develop updates into networking.

Reply:
Yes, I re-implemented fill here (although I made a small fix to it that only matters if the given length is shorter than the length of the values vector.

Reply:
I was not sure if merging develop into this is a good idea since we were developing the networking branch separately and will merge this into develop later on. 

Please let me know if you would still like me to do that. Otherwise, I can handle this merge issue when merging networking to develop on the GitHub interface in a few seconds.

Reply:
@jesings Can we merge develop into networking to update?

Reply:
Ok develop is merged into networking @youngfalcon now you should merge networking into this and then remove this reimplementation.